schemas/2016-01-29/Microsoft.PowerBI.json

```diff
@@ -0,0 +1,101 @@
+{
+  "id": "http://schema.management.azure.com/schemas/2016-01-29/Microsoft.PowerBI.json",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Microsoft.PowerBI",
+  "description": "Microsoft Power BI Resource Types",
+  "resourceDefinitions": {
+    "workspaceCollections": {
+      "description": "Microsoft.PowerBI/workspaceCollections resource",
+      "type": "object",
+      "properties": {
+        "apiVersion": {
+          "enum": [
+            "2016-01-29"
+          ]
+        },
+        "type": {
+          "enum": [
+            "Microsoft.PowerBI/workspaceCollections"
+          ]
+        },
+        "name": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^(?:[a-zA-Z0-9]+-?)+$",
+              "minLength": 3,
+              "maxLength": 63
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "location": {
+          "oneOf": [
+            {
+              "type": "string",
+              "pattern": "^[a-zA-Z0-9\\s]+$",
+              "minLength": 1,
+              "maxLength": 40
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "tags": {
+          "oneOf": [
+            {
+              "type": "object",
+              "patternProperties": {
+                "^[^<>*%&:\\\\?/+]{1,256}$": {
+                  "type": "string",
+                  "pattern": "^.{1,256}$"
+                }
+              },
+              "additionalProperties": false
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        },
+        "sku": {
+          "oneOf": [
+            {
+              "type": "object",
+              "properties": {
+                "name": {
+                  "enum": [
+                    "S1",
+                    "s1"
+                  ]
+                },
+                "tier": {
+                  "enum": [
+                    "Standard",
+                    "standard"
+                  ]
+                }
+              },
+              "required": [
+                "name"
+              ]
+            },
+            {
+              "$ref": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+            }
+          ]
+        }
+      },
+      "required": [
+        "apiVersion",
+        "type",
+        "name",
+        "location",
```

Comment:
please keep "location" in the required properties array if it is required for the resource type, even removing the property in the schema, because it has been already defined in root schema, but it is not required in the root schema because not all RP resource types require it.